### PR TITLE
add machine file for tigercpu at Princeton

### PR DESCRIPTION
**Pull request by HerculesJack**

### CODE REVIEW

**Review by jwise77 — APPROVED** [review]

File: src/enzo/Make.mach.tigercpu [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+#               cd ../ring/
+#               make
+
+#-----------------------------------------------------------------------
+# Install paths (local variables)
+#-----------------------------------------------------------------------
+
+LOCAL_COMPILER = ${PROD_DIR}/compiler
+LOCAL_MPI_INSTALL = ${MPI_HOME}
+LOCAL_HDF5_INSTALL = ${HDF5DIR}
```
Comment (clairekope) [inline src/enzo/Make.mach.tigercpu:40]: If any combination of these are defined by the user (e.g. in the `.bashrc`), and not when a module is loaded, I would suggest putting the full path here, substituting system variables like `$USER` as appropriate.

**Review by clairekope — CHANGES_REQUESTED** [review]
This file looks good, but I want confirmation that file paths such as `$HDF5DIR` will be available to any Tiger user and aren't specific to He Jia's shell environment.

File: src/enzo/Make.mach.tigercpu (hunk 1 above)
Comment (HerculesJack) [reply src/enzo/Make.mach.tigercpu:40]: Hi, I believe those paths are defined when the corresponding modules are loaded.

Comment (clairekope) [reply src/enzo/Make.mach.tigercpu:40]: Excellent! I'll remove the change request and approve.

**Review by clairekope — APPROVED** [review]

File: src/enzo/Make.mach.tigercpu (hunk 1 above)
Comment (peeples) [reply src/enzo/Make.mach.tigercpu:40]: Why all the mercurial? Neither enzo nor yt use anymore. (And yt should not be required for installing enzo.)

**Pull request merged**